### ISSUE TITLE
Improvements to post on tallow

### ISSUE THREAD

**Issue by puneetse** [clearlinux/clear-linux-documentation#457]
Regarding the documentation on tallow in this blog post: https://clearlinux.org/blogs/my-clear-linux-system-just-stopped-responding-ssh 

There are 2 improvements I think that can be made:

1. Discovering which hosts are blocked. The blog says to use the `iptables -L` command to discover whether a host is blocked or not. This isn't too useful since the command will always show the static entry, which has an ipset named *tallow* like this: `DROP       all  --  anywhere        anywhere             match-set tallow src`. A better command to show is the tallow ipset list with: `ipset list tallow`. This will output the list of blocked IP addresses and/or ranged.

2. Unblocking a blocked host. The blog post currently recommends adding a whitelist to the tallow.conf. This is good information, however it does not immediately unblock a blocked host. Adding a whitelist only prevents the host from being blocked again in the future (after the 1 hour wait period is over). It would be nice to have an immediate solution, which is `ipset del [IP ADDRESS OR RANGE]` to get readers back into their systems without waiting.

**Comment by puneetse** [comment]
@ahkok is there a better way to query and modify the tallow block list without being compiled with debug? This would be good info the man page too.

**Comment by mvincerx** [comment]
I agree that `iptables -L` is unhelpful and should be replaced. @pixelgeek, Can you share your thoughts? 

Re #1: 
@puneetse when I run your `sudo ipset list tallow`, I receive:
 "ipset v7.1: The set with the given name does not exist."
Does this mean tallow has NOT blocked my host? Or something else? 
Regardless, this needs clarification to be useful to readers.  (I've used parts of this blog post many times to troubleshoot SSH issues.  But the purpose of `iptables -L` baffled me. )

Re#2: 
Completely agree on this, but I'm not sure what rules are assigned. @phmccarty Can you shed any light?

**Comment by ahkok** [comment]
`tallow` doesn't setup `ipset` and `iptables` unless there's a reason to. It doesn't actually create the `ipset` until a host needs to be blocked.

If you receive the "set ... does not exist" it means nothing needed blocking until now.

In the case of tallow, using `iptables -L` is completely wrong and won't show the current block list.

Using `ipset delete tallow $IP` is entirely reasonable, and should probably be in the instructions.

**Comment by ahkok** [comment]
FTR `ipset list tallow` is the correct way to query the current block list (for ipv4).

**Comment by mvincerx** [comment]
@pixelgeek, you are the author of this blog post. Will you please respond here? Do you want to revise your blog post? Or would you like me to revise it per suggestions above?

**Comment by pixelgeek** [comment]
https://clearlinux.org/blogs/my-clear-linux-system-just-stopped-responding-ssh has been updated.  Note - the template tallow.conf is currently missing, but @ahkok is working on getting that added back.
